HomeWork_One/Quiz_Template/src/components/Result_Components/Statistic.tsx

```diff
@@ -0,0 +1,38 @@
+import style from "./Statistic.module.css";
+
+import { useContext } from "react";
+import { SettingsContext } from "../Context.tsx";
+import { useSelector } from "react-redux";
+import { selectQuestions } from "../../redux/questions/questionsSlice.tsx";
+import { selectGameInfo } from "../../redux/gameInfo/gameInfoSlice.ts";
+
+export function Statistic() {
+  const numberOfQuestions = useSelector(selectQuestions).questions.length;
+  const gameInfoOptions = useSelector(selectGameInfo);
+  const { statisticOption } = useContext(SettingsContext);
+  const gameInfo = useSelector(selectGameInfo).initTime.payload;
+
+  function checkTime() {
+    const initTimeInSec =
+      +gameInfo.hour * 3600 + +gameInfo.minute * 60 + +gameInfo.second;
+    const currentTimeInSec =
+      Number(new Date().getHours() * 3600) +
+      Number(new Date().getMinutes() * 60) +
+      Number(new Date().getSeconds());
+    const seconds = (currentTimeInSec - initTimeInSec) % 60;
+    const minutes = Math.floor((currentTimeInSec - initTimeInSec) / 60);
+    return `${minutes} minute ${seconds} seconds`;
```

Comment:
this is a duplication
I think you may extract this into a new utility common function or a new custom hook